components/alert/demo/banner.md

```diff
@@ -18,7 +18,7 @@ Display Alert as a banner at top of page.
 import { Alert } from 'antd';
 
 ReactDOM.render(
-  <div>
+  <>
```

Comment:
- There is no need to modify demo like that. It's better to make single PR minimal.